app/model/persisted/eventschedules/VenueType.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package model.persisted.eventschedules
+
+import play.api.libs.json._
+import reactivemongo.bson.{ BSON, BSONHandler, BSONString }
+
+object VenueType extends Enumeration {
+  type VenueType = Value
+
+  val LONDON_FSAC, NEWCASTLE_FSAC, NEWCASTLE_LONGBENTON, VIRTUAL = Value
```

Comment:
In the short term we'll keep these as Venues but they are not really (at least how we decided to think about them). We'll revisit this later.